package.json

```diff
@@ -11,7 +11,8 @@
     "start": "npm run build && node ./src/server/bin/server",
     "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
     "server": "nodemon ./src/server/bin/server",
-    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "json": "cd ./src/external-api/ && node server.js",
+    "dev": "concurrently \"npm run server\" \"npm run client\" \"npm run json\"",
```

Comment:
Please call the task `json-server` instead of just `json`. Or you could call it `external-api` if you prefer.